registration/src/main/proto/lobby/registration/process.proto

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2015, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package lobby.registration;
+
+option java_package="lobby.registration";
+option java_multiple_files = true;
+
+// The registration process state of the Process Manager.
+message RegistrationProcess {
+    // The unique process id.
+    ProcessId id = 1;
+
+    // The current state of the process.
+    State state = 2;
+
+    // The state of the process.
+    enum State {
+        // The initial state of the process.
+        NOT_STARTED = 0;
+
+        // The process is in this state when an order is placed by a registrant but the reservation is not confirmed yet.
+        AWAITING_RESERVATION_CONFIRMATION = 1;
+
+        // The process is in this state when the system has temporarily reserved the seats requested by the registrant.
+        RESERVATION_CONFIRMED = 2;
+
+        // The process is in this state when the payment for the order is received.
+        PAYMENT_RECEIVED = 3;
+
+        // The process is in this state when it is completed:
+        // the reservation was confirmed (for free-of-charge orders) or
+        // the payment is received (for billable orders).
+        COMPLETED = 4;
+    }
+}
+
+// The id of the registration process based on the UUID.
```

Comment:
Do not mention UUID in the description of the type. It's implementation details.